Auto update failing for zsync

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [zsync](https://github.com/termux/termux-packages/blob/d426dd078e76b8d858707e3023e67be3346bad70/packages/zsync) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating zsync [Current version: 0.6.3]
INFO: package being updated to 0.6.4.
Downloading https://zsync.moria.org.uk/download/zsync-0.6.4.tar.bz2
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 11  269k   11 31612    0     0  29008      0  0:00:09  0:00:01  0:00:08 29001
100  269k  100  269k    0     0   188k      0  0:00:01  0:00:01 --:--:--  188k

You are about to commit these changes:

--------------------
diff --git a/packages/zsync/build.sh b/packages/zsync/build.sh
index 9c30eda91d..705545a40c 100644
--- a/packages/zsync/build.sh
+++ b/packages/zsync/build.sh
@@ -2,9 +2,8 @@ TERMUX_PKG_HOMEPAGE=https://zsync.moria.org.uk/
 TERMUX_PKG_DESCRIPTION="A file transfer program to download files from remote web servers"
 TERMUX_PKG_LICENSE="Artistic-License-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.6.3"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="0.6.4"
 TERMUX_PKG_SRCURL=https://zsync.moria.org.uk/download/zsync-${TERMUX_PKG_VERSION}.tar.bz2
-TERMUX_PKG_SHA256=293b6191821641d3ed6248206f8f9df0bf46e6ee2cf8b4dd97cfd1d5909edb9a
+TERMUX_PKG_SHA256=f1d6d3e8e79933e9e03dd9f342673f31274686a29d295e7cb34558755d224670
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
--------------------

bump(main/zsync): 0.6.4


Not committing to Git!

INFO: Trying to build package.
INFO: Package zsync exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
Loading restricted AppArmor profile...
termux - building zsync for arch aarch64...
::group::INFO: Fetching repo metadata
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100 13134  100 13134    0     0  60881      0 --:--:-- --:--:-- --:--:-- 60805
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100 13320  100 13320    0     0  53010      0 --:--:-- --:--:-- --:--:-- 52857
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100 13120  100 13120    0     0  61739      0 --:--:-- --:--:-- --:--:-- 61596
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100   833  100   833    0     0   3918      0 --:--:-- --:--:-- --:--:--  3929
[packages-cf.termux.dev-apt-termux-x11]: gpg: Signature made Fri 15 May 2026 06:24:34 AM UTC
[packages-cf.termux.dev-apt-termux-x11]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-x11]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100   833  100   833    0     0   3042      0 --:--:-- --:--:-- --:--:--  3051
[packages-cf.termux.dev-apt-termux-main]: gpg: Signature made Fri 15 May 2026 03:58:34 PM UTC
[packages-cf.termux.dev-apt-termux-main]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-main]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-main]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100   833  100   833    0     0   2764      0 --:--:-- --:--:-- --:--:--  2767
[packages-cf.termux.dev-apt-termux-root]: gpg: Signature made Tue 12 May 2026 05:30:27 PM UTC
[packages-cf.termux.dev-apt-termux-root]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-root]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
::endgroup::
Downloading zsync@0.6.4 source from 'https://zsync.moria.org.uk/download/zsync-0.6.4.tar.bz2' if necessary...
Downloading https://zsync.moria.org.uk/download/zsync-0.6.4.tar.bz2
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 61  269k   61  166k    0     0   127k      0  0:00:02  0:00:01  0:00:01  127k
100  269k  100  269k    0     0   194k      0  0:00:01  0:00:01 --:--:--  193k
configure: WARNING: unrecognized options: --disable-rpath, --disable-rpath-hack, --disable-nls, --enable-shared, --enable-static
checking build system type... x86_64-pc-linux-gnu
checking host system type... aarch64-unknown-linux-android
checking for a BSD-compatible install... /usr/bin/install -c
checking whether sleep supports fractional seconds... yes
checking filesystem timestamp resolution... 0.01
checking whether build environment is sane... yes
/home/builder/.termux-build/zsync/src/autotools/missing: Unknown `--is-lightweight' option
Try `/home/builder/.termux-build/zsync/src/autotools/missing --help' for more information
configure: WARNING: 'missing' script is too old or missing
checking for aarch64-linux-android-strip... llvm-strip
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking xargs -n works... yes
checking whether UID '1001' is supported by ustar format... yes
checking whether GID '1001' is supported by ustar format... yes
checking how to create a ustar tar archive... gnutar
checking whether to enable maintainer-specific portions of Makefiles... no
checking for aarch64-linux-android-gcc... aarch64-linux-android-clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether aarch64-linux-android-clang accepts -g... yes
checking for aarch64-linux-android-clang option to enable C23 features... -std=gnu23
checking whether aarch64-linux-android-clang -std=gnu23 understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of aarch64-linux-android-clang -std=gnu23... none
checking whether ln -s works... yes
checking for aarch64-linux-android-ranlib... llvm-ranlib
checking for an ANSI C-conforming const... yes
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for string.h... (cached) yes
checking for size_t... yes
checking for memcpy... yes
checking for pwrite... yes
checking for pread... yes
checking for mkstemp... yes
checking for socklen_t... yes
checking for in_port_t... yes
checking for h_errno declaration in netdb.h... yes
checking for socket in -lsocket... no
checking for getaddrinfo... yes
checking for aarch64-linux-android-clang -std=gnu23 option to support large files... none needed
checking for declarations of fseeko and ftello... yes
checking size of size_t... 8
checking size of off_t... 8
checking if malloc debugging is wanted... no
checking whether compiler supports -g... no
checking whether compiler supports -Wall... no
checking whether compiler supports -Wwrite-strings... no
checking whether compiler supports -Winline... no
checking whether compiler supports -Wextra... no
checking whether compiler supports -Winline... no
checking whether compiler supports -Wmissing-noreturn... no
checking whether compiler supports -Wredundant-decls... no
checking whether compiler supports -Wnested-externs... no
checking whether compiler supports -Wundef... no
checking whether compiler supports -Wbad-function-cast... no
checking whether compiler supports -Wcast-align... no
checking whether compiler supports -Wvolatile-register-var... no
checking whether compiler supports -ffast-math... no
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating librcksum/Makefile
config.status: creating zlib/Makefile
config.status: creating libzsync/Makefile
config.status: creating doc/Makefile
config.status: creating config.h
config.status: executing depfiles commands
configure: WARNING: unrecognized options: --disable-rpath, --disable-rpath-hack, --disable-nls, --enable-shared, --enable-static
make  all-recursive
make[1]: Entering directory '/home/builder/.termux-build/zsync/src'
Making all in librcksum
make[2]: Entering directory '/home/builder/.termux-build/zsync/src/librcksum'
aarch64-linux-android-clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..   -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o md4test.o md4test.c
aarch64-linux-android-clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..   -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o md4.o md4.c
aarch64-linux-android-clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..   -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o rsum.o rsum.c
aarch64-linux-android-clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..   -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o rsumtest.o rsumtest.c
aarch64-linux-android-clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..   -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o hash.o hash.c
aarch64-linux-android-clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..   -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o range.o range.c
aarch64-linux-android-clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..   -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o state.o state.c
aarch64-linux-android-clang -std=gnu23  -fstack-protector-strong -Oz  -L/data/data/com.termux/files/usr/lib -Wl,-rpath=/data/data/com.termux/files/usr/lib -Wl,--enable-new-dtags -Wl,--as-needed -Wl,-z,relro,-z,now -o md4test md4test.o md4.o  
aarch64-linux-android-clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..   -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o ../progress.o ../progress.c
rm -f librcksum.a
ar cr librcksum.a rsum.o hash.o state.o range.o md4.o 
llvm-ranlib librcksum.a
aarch64-linux-android-clang -std=gnu23  -fstack-protector-strong -Oz  -L/data/data/com.termux/files/usr/lib -Wl,-rpath=/data/data/com.termux/files/usr/lib -Wl,--enable-new-dtags -Wl,--as-needed -Wl,-z,relro,-z,now -o rsumtest rsum.o rsumtest.o hash.o range.o state.o md4.o ../progress.o  
make[2]: Leaving directory '/home/builder/.termux-build/zsync/src/librcksum'
Making all in zlib
make[2]: Entering directory '/home/builder/.termux-build/zsync/src/zlib'
aarch64-linux-android-clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..   -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o inflate.o inflate.c
aarch64-linux-android-clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..   -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o adler32.o adler32.c
aarch64-linux-android-clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..   -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o inftrees.o inftrees.c
aarch64-linux-android-clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..   -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o zutil.o zutil.c
adler32.c:47:23: error: unknown type name 'adler'
   47 | uLong ZEXPORT adler32(adler, buf, len)
      |                       ^
adler32.c:47:30: error: unknown type name 'buf'
   47 | uLong ZEXPORT adler32(adler, buf, len)
      |                              ^
adler32.c:47:35: error: unknown type name 'len'
   47 | uLong ZEXPORT adler32(adler, buf, len)
      |                                   ^
adler32.c:47:39: error: expected ';' after top level declarator
   47 | uLong ZEXPORT adler32(adler, buf, len)
      |                                       ^
      |                                       ;
adler32.c:51:1: error: expected identifier or '('
   51 | {
      | ^
5 errors generated.
make[2]: *** [Makefile:363: adler32.o] Error 1
make[2]: *** Waiting for unfinished jobs....
inflate.c:107:26: error: unknown type name 'strm'
  107 | int ZEXPORT inflateReset(strm)
      |                          ^
inflate.c:107:31: error: expected ';' after top level declarator
  107 | int ZEXPORT inflateReset(strm)
      |                               ^
      |                               ;
inflate.c:109:1: error: expected identifier or '('
  109 | {
      | ^
inflate.c:128:27: error: unknown type name 'strm'
  128 | int ZEXPORT inflateInit2_(strm, windowBits, version, stream_size)
      |                           ^
inflate.c:128:33: error: unknown type name 'windowBits'
  128 | int ZEXPORT inflateInit2_(strm, windowBits, version, stream_size)
      |                                 ^
inflate.c:128:45: error: unknown type name 'version'
  128 | int ZEXPORT inflateInit2_(strm, windowBits, version, stream_size)
      |                                             ^
inflate.c:128:54: error: unknown type name 'stream_size'
  128 | int ZEXPORT inflateInit2_(strm, windowBits, version, stream_size)
      |                                                      ^
inflate.c:128:66: error: expected ';' after top level declarator
  128 | int ZEXPORT inflateInit2_(strm, windowBits, version, stream_size)
      |                                                                  ^
      |                                                                  ;
inflate.c:133:1: error: expected identifier or '('
  133 | {
      | ^
inflate.c:171:26: error: unknown type name 'strm'
  171 | int ZEXPORT inflateInit_(strm, version, stream_size)
      |                          ^
inflate.c:171:32: error: unknown type name 'version'
  171 | int ZEXPORT inflateInit_(strm, version, stream_size)
      |                                ^
inflate.c:171:41: error: unknown type name 'stream_size'
  171 | int ZEXPORT inflateInit_(strm, version, stream_size)
      |                                         ^
inflate.c:171:53: error: expected ';' after top level declarator
  171 | int ZEXPORT inflateInit_(strm, version, stream_size)
      |                                                     ^
      |                                                     ;
inflate.c:175:1: error: expected identifier or '('
  175 | {
      | ^
inflate.c:189:24: error: unknown type name 'state'
  189 | inftrees.c:32:19: error: unknown type name 'type'
l   32 | int inflate_tablocea(lt yvpoei,d  lfeinxse,d tcaobdleess,( sttaabtlee),
       b| i                       ^t
s, work)
      |                   ^
inflate.c:189:30: error: expected ';' after top level declarator
  189 | local void fixedtables(state)
      |                              ^
      |                              ;
inflate.c:191:1: error: expected identifier or '('
  191 | {
      | ^
inftrees.c:32:25: error: unknown type name 'lens'
   32 | int inflate_table(type, lens, codes, table, bits, work)
      |                         ^
inflate.c:307:18: error: unknown type name 'strm'
  307 | int updatewindow(strm, out)
      |                  ^
inflate.c:307:24: error: unknown type name 'out'
  307 | int updatewindow(strm, out)
      |                        ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
inftrees.c:32:31: error: unknown type name 'codes'; did you mean 'code'?
   zutil.c:137:29: error: unknown type name 'err'
32  137 | const char * ZEXPORT zError(err)
      |                             ^
 | int inflate_table(zutil.c:137:33: error: expected ';' after top level declarator
  t137 | const char * ZEXPORT zError(err)y
      p|                                 ^
      |                                 ;
ezutil.c:139:1: error: expected identifier or '('
  139 | {
      | ^
, len20s, codes, table, bits, work)
      |                               ^~~~~
      |                               code
./inftrees.h errors generated.
:28:3: note: 'code' declared here
   28 | } code;
      |   ^
zutil.c:301:17: error: unknown type name 'opaque'
  301 | voidpf zcalloc (opaque, items, size)
      |                 ^
inftrees.c:32:38: error: unknown type name 'table'
zutil.c   :301:25: error: unknown type name 'items'
  301 | voidpf zcalloc (opaque, items, size)
      |                         ^
32 | int inflate_tabzutil.c:301:32: error: unknown type name 'size'
  301 | voidpf zcalloc (opaque, items, size)
      |                                ^
lezutil.c:301:37: error: expected ';' after top level declarator
  301 | voidpf zcalloc (opaque, items, size)
      |                                     ^
      |                                     ;
(zutil.c:305:1: error: expected identifier or '('
  305 | {
      | ^
type, lens, codes, tablzutil.c:311:15: error: unknown type name 'opaque'
  311 | void  zcfer,ee (opaque, ptr)
      |               ^
 bits, work)
      | zutil.c:311:23: error: unknown type name 'ptr'
  311 | void  zcfree (opa                                     ^que, ptr)
      |                       ^

zutil.c:311:27: error: expected ';' after top level declarator
  311 | void  zcfree (opaque, ptr)
      |                           ^
      |                           ;
zutil.c:314:1: error: expected identifier or '('
  314 | {
      | ^
inftrees.c:32:45: error: unknown type name 'bits'
   32 | int inflate_table(type, lens, codes, table, bits, work)
      |                                             ^
12 errorsinftrees.c generated.
:32:51: error: unknown type name 'work'
   32 | int inflate_tablmake[2]: *** [Makefile:363: inflate.o] Error 1
e(type, lens, codes, table, bits, work)
      |                                                   ^
inftrees.c:32:56: error: expected ';' after top level declarator
   32 | int inflate_table(type, lens, codes, table, bits, work)
      |                                                        ^
      |                                                        ;
inftrees.c:39:1: error: expected identifier or '('
   39 | {
      | ^
8 errors generated.
make[2]: *** [Makefile:363: zutil.o] Error 1
make[2]: *** [Makefile:363: inftrees.o] Error 1
make[2]: Leaving directory '/home/builder/.termux-build/zsync/src/zlib'
make[1]: *** [Makefile:760: all-recursive] Error 1
make[1]: Leaving directory '/home/builder/.termux-build/zsync/src'
make: *** [Makefile:594: all] Error 2
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occurred when I last tried to update at 2026-05-15 17:43:02 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/25918008525">25918008525</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/zsync/build.sh
+++ b/packages/zsync/build.sh
@@ -2,9 +2,13 @@ TERMUX_PKG_HOMEPAGE=https://zsync.moria.org.uk/
 TERMUX_PKG_DESCRIPTION="A file transfer program to download files from remote web servers"
 TERMUX_PKG_LICENSE="Artistic-License-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.6.3"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="0.6.4"
 TERMUX_PKG_SRCURL=https://zsync.moria.org.uk/download/zsync-${TERMUX_PKG_VERSION}.tar.bz2
-TERMUX_PKG_SHA256=293b6191821641d3ed6248206f8f9df0bf46e6ee2cf8b4dd97cfd1d5909edb9a
+TERMUX_PKG_SHA256=f1d6d3e8e79933e9e03dd9f342673f31274686a29d295e7cb34558755d224670
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_pre_configure() {
+	# Workaround compiler error due to older C syntax
+	CFLAGS+=" -std=c11"
+}

